### DISCUSSION TITLE
Connecting to existing Fluent session, password?

### DISCUSSION THREAD

**Discussion by hanchak** [ansys/pyfluent#1031] [General]
I used the following to try to connect to my existing Fluent session:

`solver = pyfluent.launch_fluent(start_instance=False, ip='127.0.0.1', port='1055')`

I received the following error:

`RuntimeError: Fluent session password mismatch`

Is there a default password to connect to Fluent?

Thanks,
Mike

**Comment by hanchak** [comment]
Forgot to mention that a search of "issues" mentioned logging into a different machine. In my case, I am on the same machine as the fluent, local license, and Windows 10.

**Comment by dnwillia-work** [comment]
@mkundu1 What might be going on here?

**Comment by mkundu1** [comment]
@hanchak In the latest PyFluent release, the following command will connect to a running Fluent server:
```
solver = pyfluent.launch_fluent(start_instance=False, server_info_filepath="server.txt")
```
You need to first launch Fluent with the server. The easiest way of doing that is to check the "Start Server" option in Fluent Launcher > General Options. Fluent will write out the server info file. Please check Fluent documentation for additional ways of starting the Fluent server.

**Reply by hanchak** [reply]
This worked. thank you.
